components/support/rc_crypto/src/contentsignature.rs

```diff
@@ -0,0 +1,305 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
```

Comment:
Please include some brief high-level module docs here, even if just some links out to further reading. I fear future maintainers of this crate may not have all the context to remember what this is for.